src/constants/daysOfTheWeek.js

```diff
@@ -0,0 +1 @@
+export default ['monday', 'tuesday', 'wednesday', 'thursday', 'friday', 'saturday', 'sunday'];
```

Comment:
imo, it would be better to have a file `dates` and all dates things related to it in this file. 
Create a constants for each day, and those constants put to exported array (btw, those days also should be exported). Then, `tuesday` and `monday` in app.js should be used by those constants.